### ISSUE TITLE
Broken Links

### ISSUE THREAD

**Issue by excentrickristy** [jakartaee/specifications#520]
Hello,
I've been working with Shabnam on technical SEO cleanup and found the following issues.

On this page:
https://jakarta.ee/specifications/bean-validation/3.0/jakarta-bean-validation-spec-3.0.html

The "a" tag should be removed from these two URLs because they are registering as broken links
![image](https://user-images.githubusercontent.com/73264129/177048941-3c66327f-791b-402a-9ee6-b51722d37994.png)

---
On this page:
https://jakarta.ee/specifications/enterprise-beans/4.0/jakarta-enterprise-beans-spec-core-4.0.html

There are a number of links in the content that are meant to serve as page anchors but the URL of the page must have changed at some point. You can see highlighted here at example.
![image](https://user-images.githubusercontent.com/73264129/177049297-daf8b0dd-fd30-4dbc-b54f-f87ec963a96b.png)

On this page, there needs to be a search and replace to that any anchor tags written with this href:
./enterprise-beans-spec-opt-4.0.html

are replaced with:
./jakarta-enterprise-beans-spec-opt-4.0.html

There are also references on this page to Ejb.html which much have been a previous name for this document - they need to be replaced:
![image](https://user-images.githubusercontent.com/73264129/177049772-f021dc61-a4ff-4a91-a3b5-12f95a2c5f0b.png)

They should be replaced with this as well to fix these broken links:
./jakarta-enterprise-beans-spec-opt-4.0.html

---

On these pages:
https://jakarta.ee/specifications/cdi/4.0/apidocs/overview-tree.html
https://jakarta.ee/specifications/cdi/4.0/apidocs/allclasses-index.html
https://jakarta.ee/specifications/cdi/4.0/apidocs/allclasses.html
https://jakarta.ee/specifications/cdi/4.0/apidocs/index-all.html
https://jakarta.ee/specifications/cdi/4.0/apidocs/jakarta.cdi/jakarta/enterprise/inject/build/compatible/spi/class-use/buildservices
https://jakarta.ee/specifications/cdi/4.0/apidocs/jakarta.cdi/jakarta/enterprise/inject/build/compatible/spi/package-summary.html
https://jakarta.ee/specifications/cdi/4.0/apidocs/jakarta.cdi/jakarta/enterprise/inject/build/compatible/spi/package-tree.html

The BuildServicesResolver link is broken

Links to: https://jakarta.ee/specifications/cdi/4.0/apidocs/jakarta.cdi/jakarta/enterprise/inject/build/compatible/spi/BuildServicesResolver.html

Not sure what the correct link is though.

---

On this page:
https://jakarta.ee/specifications/faces/3.0/apidocs/com/sun/faces/application/resource/resourcehandlerimpl

There are multiple broken links to the overview summary: 
https://jakarta.ee/specifications/faces/3.0/apidocs/com/sun/overview-summary.html

Not sure what the correct link is though.

**Comment by chrisguindon** [comment]
@ivargrimstad How can we ensure that these issues are being assigned and dealt with by our spec projects?

@excentrickristy Created a few similar issues in the last 6 months but they haven't been dealt with:

- https://github.com/jakartaee/specifications/issues/583
- https://github.com/jakartaee/specifications/issues/582

Thanks!

**Comment by chrisguindon** [comment]
@chrisguindon Do you need anything from me or @excentrickristy to ensure that this is brought to the attention of the spec project?

**Comment by ivargrimstad** [comment]
I'll see what I can do. Don't expect this to be prioritized, though. It is hard enough to get releases out for new specs, so I wouldn't be too optimistic about getting out a service release of an old spec to fix a couple of links...
For the future, it would be great if @excentrickristy could create the issue directly in the affected spec project's issue tracker.

**Comment by chrisguindon** [comment]
> For the future, it would be great if @excentrickristy could create the issue directly in the affected spec project's issue track

She's looking at this from a website perspective.  What's the best way for her to find the right repo?

For example, for  [https://jakarta.ee/specifications/cdi/*](https://jakarta.ee/specifications/cdi/4.0/apidocs/overview-tree.html) is it simply a question of searching for "cdi" via the github search for the jakarta.ee organization? 

https://github.com/jakartaee?q=cdi&type=all&language=&sort=

**Comment by ivargrimstad** [comment]
You're right, it is probably a bit tricky to find the right one. It can also be in the EE4J github organization, which isn't even called EE4J, but [eclipse-ee4j](https://github.com/eclipse-ee4j) to make it more interesting :)

Creating one issue per spec (in your example "cdi") here is a step in the right direction. Then someone from the spec committee can move or copy the issue over to the correct repository. Collecting multiple findings in one issue across multiple specs makes it harder to follow up.
